phpunit/class-performance-test.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Server-side performance tests
+ *
+ * @package Gutenberg
+ */
+
+// To run these tests, set the RUN_SLOW_TESTS environment variable to a truthy
+// value.
+if ( getenv( 'RUN_SLOW_TESTS' ) ) {
+	class Performance_Test extends WP_UnitTestCase {
+		function test_parse_large_post() {
+			$start = microtime( true );
+			$start_mem = memory_get_usage();
+
+			$html = file_get_contents(
+				dirname( __FILE__ ) . '/fixtures/long-content.html'
```

Comment:
It's not clear to me how to do this in a simple way. It would require executing Node.js, much like generating the PHP parser, which is also checked into the repo. I'd prefer to leave this as-is for now.